### ISSUE TITLE
(Feature Request) Improved mobile view

### ISSUE THREAD

**Issue by BeatLink** [zadam/trilium#4335]
### Describe feature

I've created some scripts to rework how the mobile view in trillium functions. These scripts allow mobile users to use the full capabilities of the trillium web in desktop mode while still de-cluttering the interface and increasing the size of elements for larger screens. The scripts can be applied as is to current versions of trillium but perhaps it could be incorporated into the core mobile view of Trilium. 

Give it a try and let me know what you think!

https://github.com/BeatLink/trilium-scripts/tree/main/Mobile%20View

**Comment by BeatLink** [comment]
Yeah, the sidebar view is intended to work with the sidebar kept open

**Comment by LokiLuciferase** [comment]
This is exactly what I needed from the mobile view, thanks!

I think passing the note IDs used in `SetupButton.js` via relations (`api.currentNote.getRelationValue`) is cleaner than putting the actual IDs in the code, but other than that, it worked out of the box like a charm!
